cli/tests/pcluster/models/test_cluster.py

```diff
@@ -122,6 +122,25 @@ def test_tags(self, cluster, mocker, existing_tags):
             )
         ).is_true()
 
+        # Add name tag:
+        cluster_name = cluster.name
+        tags["parallelcluster:cluster-name"] = cluster_name
```

Comment:
could you move line 125-127 to before line 105, so we can avoid code duplication on line 128-129

Reply:
Done!